.github/workflows/CI.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        conda-env: [base, minimal]
+        python-version: [3.6, 3.7]
+    env:
+      PYVER: ${{ matrix.python-version }}
+      CONDA_ENV: ${{ matrix.conda-env }}
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Setup Information
+      shell: bash
+      run: |
+        uname -a
+        df -h
+        ulimit -a
+        conda --version
+
+    - name: Create Environment
+      shell: bash
+      run: |
+        eval "$(conda shell.bash hook)" && conda activate
+        python devtools/scripts/create_conda_env.py -n=test -p=$PYVER devtools/conda-envs/$CONDA_ENV.yaml
+
+    - name: Install
+      shell: bash
+      run: |
+        eval "$(conda shell.bash hook)" && conda activate test
+        python -m pip install . --no-deps
+
+    - name: Environment Information
+      shell: bash
+      run: |
+        eval "$(conda shell.bash hook)" && conda activate test
+        conda list --show-channel-urls
+
+    - name: PyTest
+      shell: bash
+      run: |
+        eval "$(conda shell.bash hook)" && conda activate test
+        pytest -v --cov=qcelemental --cov-report=xml qcelemental/tests/
```

Comment:
try adding `--color yes` for quick id of failed

Reply:
Color worked!